osu-framework/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full-vnc
+
+# Install Prerequisites
+COPY compile /tmp
+USER root
+RUN cd /tmp && /tmp/compile && rm -rf /tmp/compile
```

Comment:
Tiny comment in passing, the `rm` here won't actually remove the `compile` file from the Docker image history, as it was embedded "forever" in another layer by `COPY compile /tmp`.

While the file is too small to increase the image size significantly, a "cleaner" way to use such temporary scripts is to import + use + delete them in a single Docker command, e.g. by using `git clone`, or by `wget`ing or `curl`ing the file directly from a GitHub raw URL.

Then again, the `compile` file is relatively small: Why not embed all of its commands into the Dockerfile directly?

Reply:
Its a simple hack to allow lesser layers when the image is built, which allows faster pulls, since it is quite small I'll just add it in inline on the base image pr

Reply:
Ah, sorry, I see that you've already done that in https://github.com/gitpod-io/workspace-images/pull/49/. Sorry for the noise! :relaxed: